test_helper_schema/deploy/create_applications.sql

```diff
@@ -10,6 +10,11 @@ begin;
   $function$
     begin
 
+      -- Create initial review step rows
+      insert into ggircs_portal.review_step(step_name, is_active)
+        values ('administrative', true), ('technical', true)
+        on conflict (id) do nothing;
+
```

Comment:
Seeing how this has side effects on a number of database tests, it might be easier to add a `test_helper_schema.create_review_steps` function and to call it in the two e2e tests fixtures where it is needed.

Reply:
On second thoughts, inserting this data in `data/schema/prod` may be a better option. 
Also, that `on conflict (id)` is useless because we're not inserting any id explicitly in this query.

My recommendation here would be to add a `data/schema/prod/review_step.sql`  file, with the following: 
```sql
begin;

insert into ggircs_portal.review_step(step_name, is_active)
  values ('administrative', true), ('technical', true)
  on conflict(step_name) do update set
  is_active=excluded.is_active;

commit;
```

and to add an entry in `schema/data/deploy-data.sh`, in the `deployProdData()` function

Reply:
In summary, this solution 23d095b appears not to be enough to get the review_step seeds into the tests. Perhaps in the morning @dleard you can advise us?

I've tried removing the disabling of triggers in `fixtures/analyst-all-access-setup` with no luck.

These seeds are relevant specifically for:
- the email spec, because you can't click "make decision" unless all review steps are complete, and
- the analyst accessibility test (which isn't failing, but also isn't realistic if there are no review steps)